src/external/VTK.jl

```diff
@@ -0,0 +1,58 @@
+import WriteVTK: vtk_grid, vtk_save
+
+# Uses WriteVTK.jl to convert scfres structure to VTk file format
+
+"""
+    save_scfres(filename, scfres, Val(:vtk))
+
+The function takes in the VTK filename and the scfres structure and stores into a VTK file.
+
+Grid Values:
+
+- ρ -> Density in real space
+- ψreal\\_i\\_j -> Real values of Bloch waves in real space, where i and j are Kpoint and EigenVector indexes respectively
+- ψimag\\_i\\_j -> Imaginary values of Bloch waves in real space, where i and j are Kpoint and EigenVector indexes respectively
+- ρspin -> Real value of ρspin are stored if ρspin in present
+
+MetaData:
+Energy , EigenValues, Fermi Level and occupation.
+"""
+function save_scfres(filename::AbstractString, scfres::NamedTuple, format::Val{:vtk})
+    # Initialzing the VTK Grid
+    basis = scfres.basis
+    grid = zeros(3, basis.fft_size...)
+    for (idcs, r) in zip(CartesianIndices(basis.fft_size), r_vectors_cart(basis))
+        grid[:, Tuple(idcs)...] = r
+    end
+    vtkfile = vtk_grid(filename, grid)
+    
+    # Storing the Bloch Waves in Real space
+    for i in 1:length(basis.kpoints)
+        for j in 1:size(scfres.ψ[1])[2]
+            vtkfile["ψ_real_$(i)_$(j)"] = real.(G_to_r(basis, basis.kpoints[i], scfres.ψ[i][:,j]))
```

Comment:
Can you add a `k` and `band` or something to the label. Otherwise you always have to remember which index is the band and which is the kpoint.